sgt-back-end/index.js

```diff
@@ -0,0 +1,157 @@
+const express = require('express');
+const app = express();
+const pg = require('pg');
+
+// only create ONE pool for your whole server
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.use(express.json());
+app.get('/api/grades', function (req, res) {
+  // eslint-disable-next-line no-console
+  const sql = `
+    select *
+      from "grades"
+  `;
+  db.query(sql)
+    .then(result => {
+      res.status(200).res.json(result.rows);
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'An unexpected error occurred.'
+      });
+    });
+
+});
+
+app.post('/api/grades', function (req, res) {
+  const data = req.body;
+  if (!data.score || (data.score < 0 || data.score > 100)) {
+    res.status(400).json({
+      error: 'Invalid score'
+    });
+    return;
+  } else if (!data.course || !data.name || !data.gradeId) {
+    res.status(400).json({
+      error: 'Invalid gradeId, course or name'
+    });
+    return;
+  }
+
+  // eslint-disable-next-line no-console
+  const sql = `
+   insert into "grades" ("gradeId", "name", "course", "score")
+   values ($1, $2, $3, $4)
+   returning *;
+  `;
+  const params = [data.gradeId, data.name, data.course, data.score];
+  db.query(sql, params)
+    .then(result => {
+
+      res.status(201).json(result.rows);
```

Comment:
The client should just receive an object instead of an array, so you can send back `result.rows[0]`